core/modules/qproc/testQueryAna.h

```diff
@@ -108,6 +104,8 @@ std::string buildFirstParallelQuery(QuerySession& qs, bool withSubChunks=true) {
     return first.queries[0];
 }
 
+} // anonymous namespace
```

Comment:
So, this .h file is simply included into a bunch of unit tests which are linked into separate executables.  Probably just losing the namespace entirely here is reasonable.

The other alternative would be to separate the implementations of these helpers from their decls and put the into a separate .cc which is #included into each of the tests (not sure it is worth it).